### PR TITLE
Tb/05 12 19/add webhook post storyblok

### PR DESCRIPTION
**Pull request by tanvibhakta**

### CODE REVIEW

**Comment by tanvibhakta** [comment]
Let's keep this branch open until I'm done with the feature; but please keep merging into master when you can.

File: functions/update-storyblok-gitbook.js [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
-    };
+  // Uncomment once gitbook webhook is set up
+  // if (event.httpMethod !== "POST") {
+  //   return {
+  //     statusCode: 405,
+  //     body: "Hit me with a POST please"
+  //   };
+  // }
+
+  const SBread = new StoryblokClient({
```
Comment (meetbhatt23) [inline functions/update-storyblok-gitbook.js]: Nitpicking, but might make sense to capitalise the R here. Looks like S-Bread rn

File: functions-build/update-storyblok-gitbook.js [hunk 2]
```diff
@@ -0,0 +1,6877 @@
+!(function(t, e) {
```
Comment (meetbhatt23) [inline functions-build/update-storyblok-gitbook.js]: Should this be gitignored?

**Comment by meetbhatt23** [comment]
`functions-build/update-storyblok-gitbook.js` still exists on the diff, since it was committed before the gitignore. You'll probably need to make a commit for removing it.

**Comment by tanvibhakta** [comment]
Endpoint to the function is here: https://deploy-preview-32--not-obvious.netlify.com/.netlify/functions/update-storyblok-gitbook

**Comment by meetbhatt23** [comment]
> Endpoint to the function is here: https://deploy-preview-32--not-obvious.netlify.com/.netlify/functions/update-storyblok-gitbook

Nice! Looks good :)

**Pull request closed without merging**